sql/item_latents.sql

```diff
@@ -2777,22 +2776,298 @@ INSERT INTO `item_latents` VALUES(14646, 29, 10, 32, 0); -- Darksday: MDB+10
 -- -------------------------------------------------------
 -- LAVALIER
 -- -------------------------------------------------------
-INSERT INTO `item_latents` VALUES (10961,60,-40,13,14); -- CHARMED:LIGHTRES
-INSERT INTO `item_latents` VALUES (10961,61,-40,13,14); -- CHARMED:DARKRES
+INSERT INTO `item_latents` VALUES(10961,60,-40,13,14); -- CHARMED:LIGHTRES
```

Comment:
I did a find and replace on that space because the rest of the file doesn't have it but I'll put them back.